pkg/asset/installconfig/kubevirt/validation.go

```diff
@@ -21,6 +23,42 @@ func Validate(ic *types.InstallConfig, client Client) error {
 	return allErrs.ToAggregate()
 }
 
+// ValidatePermissions tests that the current user has the required permissions
+// Some permissions are required for the installation
+// In addition, the current user is used for accessing the kubevirt/platfrom cluster from tenant cluster. E.g. list VMIs
+func ValidatePermissions(client Client, ic *types.InstallConfig) error {
+	// Prepare requests for permissions check
+	reviewObjs := createReviewObjs(ic.Platform.Kubevirt.Namespace)
+
+	// Collection of missing permissions
+	var notAllowedObjs []*authv1.SelfSubjectAccessReview
+
+	// Test each permission
+	for _, reviewObj := range reviewObjs {
+		reviewObjPointer, err := client.CreateSelfSubjectAccessReview(context.Background(), &reviewObj)
+		if err != nil {
+			return err
+		}
+
+		if !reviewObjPointer.Status.Allowed {
+			notAllowedObjs = append(notAllowedObjs, reviewObjPointer)
+		}
+	}
+
+	// Put all missing permissions in one error message
+	if len(notAllowedObjs) > 0 {
+		errMsg := "The user is missing the following permissions: "
+
+		for _, obj := range notAllowedObjs {
+			errMsg += fmt.Sprintf("%+v, ", *obj.Spec.ResourceAttributes)
+		}
+
+		return errors.New(errMsg)
```

Comment:
Use `strings.Join` so that you don't end up with a trailing comma.
```suggestion
		var notAllowed []string
		for _, obj := range notAllowedObjs {
			notAllowed = append(notAllowed, fmt.Sprintf("%+v", *obj.Spec.ResourceAttributes))
		}
		
		return fmt.Errorf("The user is missing the following permissions: %s", strings.Join(notAllowed, ", "))
```